docs/ProfilingNonJavaApplications.md

```diff
@@ -0,0 +1,36 @@
+# Profiling Non-Java Applications
+
+The scope of profiling non-java applications is limited to the case when profiler is controlled
```

Comment:
Not only. There are two ways to use async-profiler for non-Java apps: C API and LD_PRELOAD. It's worth mentioning that dynamic attach (like `asprof` does for Java) is not supported.

Reply:
LD_PRELOAD is mentioned but I do agree it is missed in the intro part of this doc